main.go

```diff
@@ -0,0 +1,209 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/http/cookiejar"
+	"os"
+	"time"
+
+	"github.com/xuri/excelize/v2"
+)
+
+const (
+	thirtyDays  int = 30
+	rowOffset   int = 1
+	sheetOffset int = 1
+)
+
+type Input struct {
```

Comment:
I find it a little confusing how there is a bunch of different pieces of information about this file, pertaining to two different sheets, stored in different locations:
- The file:
  - filename (in `Input`)
- The automated sheet:
  - sheet name (a `const` in `excel.go`)
  - sheet password (in `Input`)
  - column offsets (`const`s in `excel.go` but used elsewhere)
- The password manage sheet
  - sheet name (in `Input`)
  - column header names (in `Input`)

There's also `rowOffset` which is, what? the number of header rows? and it applies to both sheets?

And there's `sheetOffset`. I can't figure out what that is. My best guess is that it has something to do with the indexing system of the coordinates.

I would suggest unifying all of these into `Input` and making it a little clearer what each value is and what sheet it applies to. Except maybe for `sheetOffset`, I'm not sure what I would recommend for that since I don't understand it.

What do you think?

Reply:
Sounds good to make this clear. 
`rowOffset` is the number of header rows. Common for all sheets.
`sheetOffset` is the offset in x, y coordinates b/c the sheet starts counting rows and columns from 1.
Good idea to put them all in input.